terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/redirect/LoginController.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright(c) 2013 NTT DATA Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package org.terasoluna.gfw.functionaltest.app.redirect;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+@RequestMapping(value = "login")
+public class LoginController {
+
+    @RequestMapping(method = RequestMethod.GET)
+    public String login() {
+        return "redirect/login";
+    }
+
+}
```

Comment:
- [ ] If you assert depend on URL, it's not need to create this controller. (Required by case of fix)

Reply:
dc863777392a74090f5c0f2489f0509dff3a6b99

I removed un-need Controller and JSP because I changed to assert URL at 5e48867be276fbb35e772124bd0b0003c6d70667 commit .